src/image/imageengine.js

```diff
@@ -70,6 +85,12 @@ export default class ImageEngine extends Plugin {
 			.consuming( { attribute: [ 'alt' ] } )
 			.toAttribute( viewImage => ( { key: 'alt', value: viewImage.getAttribute( 'alt' ) } ) );
 
+		// Build converter for srcset attribute.
+		buildViewConverter().for( data.viewToModel )
+			.from( { name: 'img', attribute: { srcset: /./ } } )
```

Comment:
Is this regexp really required by the builder? If it's meant to be a wildcard, it isn't (`.` doesn't match new lines) and matching just one character looks a bit odd in general.